middle_end/flambda/un_anf.ml

```diff
@@ -156,83 +156,100 @@ let make_var_info (clam : Clambda.ulambda) : var_info =
           ignore_debuginfo dbg;
           ignore_var_option env)
```

Comment:
The parenthesis on this line ends the function starting on line 144. As discussed above, you'll need to return the new `environment_vars`, so changing this to `ignore_var_option env; environment_vars)` should work (although as a style guideline, I strongly recommend adding a newline after each semicolon).